packages/mobile/src/components/ErrorMessageInline.tsx

```diff
@@ -42,15 +42,16 @@ type Props = DispatchProps & OwnProps & WithTranslation & StateProps
 function ErrorMessageInline(props: Props) {
   const { error, displayMethod, dismissAfter, t } = props
 
+  React.useEffect(() => {
+    const timer = window.setTimeout(props.hideAlert, (dismissAfter || DISMISS_DEFAULT) * 1000)
+    return () => window.clearTimeout(timer)
+  }, [error])
```

Comment:
it looks like dismissAfter should be included here. 

?

Reply:
will leave comment for why i am using `error`